internal/cephfs/nodeserver_test.go

```diff
@@ -143,28 +143,27 @@ func Test_setMountOptions(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		tc := tt
```

Comment:
Same as above comment.
We'll need the local in loop variable since the test will be run in parallel.